zuul.d/cisco-nxos-jobs.yaml

```diff
@@ -38,15 +38,6 @@
       ansible_test_command: network-integration
       ansible_test_integration_targets: "nxos_.*"
 
-- job:
-    name: ansible-test-units-nxos
-    parent: ansible-test-units-base
-    required-projects:
-      - name: github.com/ansible-collections/cisco.nxos
-    timeout: 3600
-    vars:
-      ansible_collections_repo: github.com/ansible-collections/cisco.nxos
-
```

Comment:
We should also be running 2.7, 3.5, 3.6, 3.7 unit testing on nxos. Is that not the case?